src/main/java/ch/njol/skript/effects/EffBroadcast.java

```diff
@@ -42,49 +42,62 @@
 		"message to all players instead of broadcasting it."})
 @Examples({"broadcast \"Welcome %player% to the server!\"",
 		"broadcast \"Woah! It's a message!\""})
-@Since("1.0")
+@Since("1.0, INSERT VERSION (send multiple times)")
 public class EffBroadcast extends Effect {
+
 	static {
-		Skript.registerEffect(EffBroadcast.class, "broadcast %strings% [(to|in) %-worlds%]");
+		Skript.registerEffect(EffBroadcast.class, "broadcast %strings% [(to|in) %-worlds%] [%number% times]");
 	}
-	
+
 	@SuppressWarnings("null")
 	private Expression<String> messages;
+
 	@Nullable
 	private Expression<World> worlds;
-	
+
+	@SuppressWarnings("null")
+	private Expression<Number> repeat;
+
 	@SuppressWarnings({"unchecked", "null"})
 	@Override
-	public boolean init(final Expression<?>[] vars, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
-		messages = (Expression<String>) vars[0];
-		worlds = (Expression<World>) vars[1];
+	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
+		messages = (Expression<String>) exprs[0];
+		worlds = (Expression<World>) exprs[1];
+		repeat = (Expression<Number>) exprs[2];
 		return true;
 	}
-	
+
 	@Override
 	public void execute(final Event e) {
+		@SuppressWarnings("null")
+		int times = repeat.getSingle(e) != null ? repeat.getSingle(e).intValue() : 1;

```

Comment:
you shouldn't call the getter more than once - it may be a very expensive expression and also may be null on the second call but not the first.